packages/mobile/src/components/composer-input/ComposerInput.tsx

```diff
@@ -324,21 +326,21 @@ export const ComposerInput = forwardRef(function ComposerInput(
         lastKeyPressMsRef.current = Date.now()
       }
 
-      if (isAutocompleteActive && selection) {
+      if (isAutocompleteActive && selectionRef.current) {
         if (key === SPACE_KEY) {
           setIsAutocompleteActive(false)
         }
 
-        if (key === BACKSPACE_KEY && selection) {
-          const cursorPosition = selection.start
+        if (key === BACKSPACE_KEY && selectionRef.current) {
+          const cursorPosition = selectionRef.current.start
           const deletedChar = value[cursorPosition - 1]
           if (deletedChar === AT_KEY) {
             setIsAutocompleteActive(false)
           }
         }
 
         const autocompleteRange = getAutocompleteRange()
-        const cursorPosition = selection.start
+        const cursorPosition = selectionRef.current.start
```

Comment:
nit: we use this value on line 335 as well, can we move this to the top of this conditional and reuse the variable?